ts/a11y/SpeechMenu.ts

```diff
@@ -0,0 +1,211 @@
+/*************************************************************
+ *
+ *  Copyright (c) 2018-2022 The MathJax Consortium
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+/**
+ * @fileoverview Clearspeak preference menu.
+ *
+ * @author v.sorge@mathjax.org (Volker Sorge)
+ */
+
+import { SpeechExplorer } from './explorer/KeyExplorer.js';
+import { ExplorerMathItem } from './explorer.js';
+import {MJContextMenu} from '../ui/menu/MJContextMenu.js';
+import {Submenu} from 'mj-context-menu/js/item_submenu.js';
+import Sre from './sre.js';
+
+/**
+ * Values for the ClearSpeak preference variables.
+ */
+let csPrefsSetting: {[pref: string]: string} = {};
+
+/**
+ * Generator of all variables for the Clearspeak Preference settings.
+ * @param {MJContextMenu} menu The current context menu.
+ * @param {string[]} prefs The preferences.
+ */
+function csPrefsVariables(menu: MJContextMenu, prefs: string[]) {
+  let srVariable = menu.pool.lookup('speechRules');
+  let previous = Sre.clearspeakPreferences.currentPreference();
```

Comment:
I had to update to SRE version 4.1.0-beta.3 to get the `currentPreference()` method to be available.  Maybe that need to be added to `package.json`?

Reply:
Updated `package.json` and `package-lock.json`